anonymous_tokens/cpp/privacy_pass/demo/rsa_bssa_public_metadata_privacy_pass_server_demo.cc

```diff
@@ -42,6 +42,31 @@ absl::Status RunDemo() {
     return rsa_private_key.status();
   }
 
+  // Get base64url encoded public key for client to use.
+  absl::StatusOr<bssl::UniquePtr<RSA>> rsa_public_key =
+      anonymous_tokens::CreatePublicKeyRSA(
+          test_rsa_public_key.n, test_rsa_public_key.e);
+  if (!rsa_public_key.ok()) {
+    return rsa_public_key.status();
+  }
+
+  // Compute RSA BSSA Public Key Token ID.
+  absl::StatusOr<std::string> public_key_der =
+      anonymous_tokens::RsaSsaPssPublicKeyToDerEncoding(
+          rsa_public_key.value().get());
+  if (!public_key_der.ok()) {
+    return public_key_der.status();
+  }
+  std::string public_key_base64URL = absl::Base64Escape(public_key_der.value());
+
+  // Replace '+' with '-' and '/' with '_'
+  // I know one could use absl::WebSafeBase64Escape instead, but it does not pad the string with '=', which is required by Privacy Pass spec.
```

Comment:
Replace with comment "// I know one could use absl::WebSafeBase64Escape instead, but it does not pad the string with '=', which is required by Privacy Pass spec." with "Using std::replace instead of absl::WebSafeBase64Escape as latter does not pad the string with '=', which is required by Privacy Pass spec".

Reply:
updated